rockPaperScissors.py

```diff
@@ -3,48 +3,51 @@
 
 import random
 
-def gameLogic(player, computer, playerScore, computerScore):
-    playerIndex = RPS.index(player)
-    computerIndex = RPS.index(computer)
-
-    gameIncrement = 0
-    if (playerIndex != computerIndex):
-        gameIncrement = 1
-        playerNextIndex = (playerIndex + 1) % len(RPS)
-        
-        if (playerNextIndex == computerIndex):
-            computerScore += 1
```

Comment:
let's not ask this function to keep track of the game score - we just want it to tell us who the winner is. This keeps this function very easy to reason about and easier to reuse. We'll manage the game score elsewhere in the next round of changes